docs/man/man1/kubectl-expose.1

```diff
@@ -27,11 +27,7 @@ re\-use the labels from the resource it exposes.
     Synonym for \-\-target\-port
 
 .PP
-\fB\-\-create\-external\-load\-balancer\fP=false
```

Comment:
It's deprecated. But we were doing the man page generation wrong. Now we are using cobra which does it right.